### ISSUE TITLE
[BUG] `AttributeError: 'ModuleInfo' object has no attribute 'handle_name_node'`

### ISSUE THREAD

**Issue by AlecHaring** [dashingsoft/pyarmor#1097]
I'm not sure how to debug what in my project is causing the error, but it seems like it occurs before any files are obfuscated.

**Executed command:** `pyarmor -d gen --enable-rft .\fake_name\ `

**Version:** `8.1.dev5`

**pyarmor.error.log**
```
17260 MainProcess 2023-04-06 12:11:37,088
Traceback (most recent call last):
  File "C:\users\alec\desktop\pythonprojects\fake_name\venv\lib\site-packages\pyarmor\cli\__main__.py", line 627, in main
    main_entry(sys.argv[1:])
  File "C:\users\alec\desktop\pythonprojects\fake_name\venv\lib\site-packages\pyarmor\cli\__main__.py", line 615, in main_entry
    args.func(ctx, args)
  File "C:\users\alec\desktop\pythonprojects\fake_name\venv\lib\site-packages\pyarmor\cli\__main__.py", line 179, in cmd_gen
    builder.process(options, pack=args.pack)
  File "C:\users\alec\desktop\pythonprojects\fake_name\venv\lib\site-packages\pyarmor\cli\generate.py", line 145, in process
    Pytransform3.pre_build(self.ctx)
  File "C:\users\alec\desktop\pythonprojects\fake_name\venv\lib\site-packages\pyarmor\cli\core\__init__.py", line 63, in pre_build
    return m.pre_build(ctx)
  File "<maker>", line 767, in pre_build
  File "<maker>", line 657, in build
  File "<maker>", line 3257, in process
  File "<maker>", line 2778, in rebuild
  File "<maker>", line 2947, in _get_module_types
  File "<maker>", line 2937, in visit
  File "<maker>", line 2937, in visit
  File "<maker>", line 2937, in visit
  [Previous line repeated 1 more time]
  File "<maker>", line 2878, in visit
AttributeError: 'ModuleInfo' object has no attribute 'handle_name_node'

```

**Comment by jondy** [comment]
Fixed in `8.1.dev6`

**Comment by AlecHaring** [comment]
Fixed
